lib/Runtime/Language/InterpreterLoop.inl

```diff
@@ -15,8 +15,14 @@
 #define INTERPRETER_OPCODE OpCode
 #define TRACING_FUNC &InterpreterStackFrame::TraceOpCode
 #endif
+
+#ifdef PROVIDE_INTERPRETER_STMTS
+#define READ_OP ReadOp_WPreviousStmtTracking<INTERPRETER_OPCODE, ByteCodeReader::ReadByteOp, TRACING_FUNC>
```

Comment:
Instead of having a separate function for this, could we add a parameter to the current `ReadOp` kind of like the tracing func where it would be a no-op in normal case and run your code for TTD's case.
I would like to keep the places where we read opcodes to a minimum.